index.html

```diff
@@ -0,0 +1,211 @@
+</head>
+
+<body>
+    <div class="topscroll" id="home"></div>
+    <header class="header easy">
+        <div class="container easy">
+            <h1>
+                <a href="#home" class="logo easy">
+                    <img src="./images/SVG/logo-gonar.svg" alt="Gonar-logo">
+                </a>
+            </h1>
+
+            <nav class="page-menu">
+                <ul class="page-menu-list ">
+                    <li><a class='easy' href="#o-firmie">O firmie</a class='easy'></li>
+            <li><a class='easy' href="#oferta">Oferta</a></li>
+                    <li><a class='easy' href="#kontakt">Kontakt</a></li>
+                </ul>
+                <button class="hamburger hamburger--squeeze" type="button">
+                <span class="info">Menu</span>
+                <span class="hamburger-box">
+                  <span class="hamburger-inner"></span>
+                </span>
+              </button>
+
+            </nav>
+        </div>
+    </header>
+    <section class="banner" id="banner">
+        <div class="container">
+            <div class="banner-content">
+                <img src="./images/SVG/oferta-banner2.svg" alt="Gonar-oferta">
+            </div>
+            <h1 class="banner-text">
+                <div class="stronger">Elewacje</div>
+                <div>Tynki</div>
+                <div>Docieplenia</div>
+                <div id="scene">
+                    <div class="layer" data-depth=".4">
+                        <div class="period shadow"><b>15 lat</b> doświadczenia</div>
+                    </div>
+
+                    <div class="layer" data-depth=".7">
+                        <div id="period" class="period"><b>15 lat</b> doświadczenia</div>
+                    </div>
+                </div>
+
+
+            </h1>
+
+
+        </div>
+    </section>
+    <section class="o-firmie" id="o-firmie">
+        <div class="container">
+            <div class="teksty fromRight">
+                <h2>O firmie</h2>
+                <p><b>Artur Gontarski</b> – jestem właścicielem firmy Gonar. W branży budowlanej pracuję od 2005r. Specjalizuję się w systemach dociepleń. Moje wieloletnie doświadczenie w tej branży zapewni Państwu kompleksową obsługę, doradztwo, wycenę
+                    i wykonawstwo.
+                </p>
+                <p>Prace elewacyjne to sztandarowy segment usług mojej firmy. Wykonujemy wszelkiego rodzaju elewacje poczynając od tradycyjnych tynków poprzez tynki cykliczne, systemy dociepleń metodą „lekko mokrą”, po systemy wentylowe z wykonaniem z ceramiki,
+                    deski elewacyjnej, betonu architektonicznego czy okładzin kamiennych. W elewacjach stosujemy również wszelkiego rodzaju elementy dekoracyjne (fasady).</p>
+            </div>
+            <div class="pict fromLeft">
+                <img class="img-level0 " src="./images/ofirmie0.jpg" alt="o nas 1">
```

Comment:
jezeli to ozdobnik to alt pusty, jesli jest tam jakas tresc to trzeba opisac w tagu alt. informacja "o nas 1" jest niezrozumiala